spec/haml_lint/tree/haml_comment_node_spec.rb

```diff
@@ -8,7 +8,70 @@
   end
 
   let(:document) { HamlLint::Document.new(normalize_indent(haml), options) }
-  subject { document.tree.find { |node| node.type == :haml_comment } }
+  let(:node) { document.tree.find { |node| node.type == :haml_comment } }
+
+  subject { node }
+
+  describe '#directives' do
+    subject { super().directives }
+
+    context 'when there are no directives' do
+      let(:haml) { '-# A comment' }
+
+      it { should == [] }
+    end
+
+    context 'when there are directives in the node' do
+      let(:haml) { '-# haml-lint:disable AltText' }
+
+      it { should eq([HamlLint::Directive.new(haml, 1, 'disable', %w[AltText])]) }
+
+      context 'with bad formatting' do
+        let(:haml) { '-#haml-lint  :disable AltText' }
```

Comment:
Ah, ignore my previous comment. I see now that you're testing different formatting here.